pkg/plugins/datasource/wrapper/datasource_plugin_wrapper_v2.go

```diff
@@ -5,15 +5,97 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/davecgh/go-spew/spew"
 	sdk "github.com/grafana/grafana-plugin-sdk-go"
+	"github.com/grafana/grafana-plugin-sdk-go/dataframe"
 	"github.com/grafana/grafana-plugin-sdk-go/genproto/datasource"
-	"github.com/grafana/grafana/pkg/components/null"
+	"github.com/grafana/grafana/pkg/bus"
 	"github.com/grafana/grafana/pkg/components/simplejson"
 	"github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/tsdb"
 )
 
+type grafanaAPI struct {
+	logger log.Logger
+}
+
+func (s *grafanaAPI) QueryDatasource(ctx context.Context, req *datasource.QueryDatasourceRequest) (*datasource.QueryDatasourceResponse, error) {
+	if len(req.Queries) == 0 {
+		return nil, fmt.Errorf("zero queries found in datasource request")
+	}
+	spew.Dump(req)
```

Comment:
nice, I didn't know about spew!
But do we still need this line or it is just left from the debugging?